Utilities/json_refactor.py

```diff
@@ -0,0 +1,93 @@
+        for i in range(length):
+            each_check_object = check_json_data[i]
+            check_object_name = each_check_object[ident_key]
+            if check_object_name in newtitle_dict:
+                ref_object_title = newtitle_dict[check_object_name]
+                if json_locale != 'en':
+                    check_object_title = each_check_object[target_key].split(dem_char)
+                    check_object_title[title_index] = ref_object_title
+                    new_check_object_title = dem_char.join(check_object_title)
+                else:
+                    new_check_object_title = ref_object_title
+                each_check_object[target_key] = new_check_object_title
+            check_json_data[i] = each_check_object
+    # Create a new JSON file with updated name
+    with open(new_json, 'w', encoding='utf-8') as new_jsonfile:
+        json.dump(check_json_data, new_jsonfile, ensure_ascii=False, indent=2)
+
+# Update the English titles in all the JSONs files
+mismatched_titles_excel = 'mismatched_titles.xlsx'
+replace_title_ref_(mismatched_titles_excel, 'original/alphabet_game_map_hi.json', 'en_updated/alphabet_game_map_hi.json', 'hi', sheet_name='alphabet')
+replace_title_ref_(mismatched_titles_excel, 'original/alphabet_game_map_en.json', 'en_updated/alphabet_game_map_en.json',  sheet_name='alphabet')
+replace_title_ref_(mismatched_titles_excel, 'original/grammar_game_map_hi.json', 'en_updated/grammar_game_map_hi.json', 'hi', sheet_name='grammar')
+replace_title_ref_(mismatched_titles_excel, 'original/grammar_game_map_en.json', 'en_updated/grammar_game_map_en.json',  sheet_name='grammar')
+replace_title_ref_(mismatched_titles_excel, 'original/shapes_game_map_hi.json', 'en_updated/shapes_game_map_hi.json', 'hi', sheet_name='shapes')
+replace_title_ref_(mismatched_titles_excel, 'original/shapes_game_map_en.json', 'en_updated/shapes_game_map_en.json',  sheet_name='shapes')
+replace_title_ref_(mismatched_titles_excel, 'original/writing_game_map_hi.json', 'en_updated/writing_game_map_hi.json', 'hi', sheet_name='writing')
+replace_title_ref_(mismatched_titles_excel, 'original/writing_game_map_en.json', 'en_updated/writing_game_map_en.json',  sheet_name='writing')
+
+# Update the Hindi titles in the Hindi JSONs files
+hindi_titles_excel = 'game_titles.xlsx'
+replace_title_ref_(hindi_titles_excel, 'en_updated/alphabet_game_map_hi.json', 'hi_updated/alphabet_game_map_hi.json', 'hi', title_index=0,  sheet_name='alphabet')
+replace_title_ref_(hindi_titles_excel, 'en_updated/grammar_game_map_hi.json', 'hi_updated/grammar_game_map_hi.json', 'hi', title_index=0, sheet_name='grammar')
+replace_title_ref_(hindi_titles_excel, 'en_updated/shapes_game_map_hi.json', 'hi_updated/shapes_game_map_hi.json', 'hi', title_index=0, sheet_name='shapes')
+replace_title_ref_(hindi_titles_excel, 'en_updated/writing_game_map_hi.json', 'hi_updated/writing_game_map_hi.json', 'hi', title_index=0, sheet_name='writing')
```

Comment:
Use for loop and append locale string to file names. Store the locale in a array. This comment is applicable from line 73 to 93.